<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="0ms"
		durationms="0"
		type="root"
		rid="1f406f5727213b8e"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="1"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		user="erdogb"
		rxversion="7.2.0.0"
		host="EC168"
		osversion="Windows 7 Service Pack 1 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="de-DE"
		dtnlanguage="de-CH"
		screenresolution="3840x1200"
		timestamp="07.03.2018 15:12:17"
		timestampiso="2018-03-07T14:12:17"
		endtime="07.03.2018 15:12:31"
		timeoutfactor="1"
		module="loginEMON">
		<detail>
			loginEMON
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:13.481"
			timeRelativeToTestModuleStartTime="00:13.481"
			timeWallClock="15:12:31"
			time="00:13.481"
			level="Error"
			category="User"
			errimg=".\loginEMO_1418ad70_4d9103fa_trace.jpg"
			errthumb=".\loginEMO_1418ad70_4d9103fa_trace_thumb.jpg">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
